Userland/Libraries/LibCore/Account.cpp

```diff
@@ -31,7 +31,7 @@ static String get_salt()
     return builder.build();
 }
 
-static Vector<gid_t> get_gids(const StringView& username)
+Vector<gid_t> Account::get_gids(const StringView& username)
```

Comment:
I think the API here should have been `Account::from_name(...)->extra_gids()` instead of exposing this internal method?